### PR TITLE
Added Out of Office description, all_day field

### PR DESCRIPTION
**Pull request by NeilAllavarpu**

### CODE REVIEW

File: account/models.py [hunk 1]
```diff
@@ -246,6 +246,8 @@ class InstructorOutOfOffice(models.Model):
     )
     start_datetime = models.DateTimeField()
     end_datetime = models.DateTimeField()
+    description = models.TextField(blank=True)
+    all_day = models.BooleanField(default=False)
```
Comment (jerrylinew) [inline account/models.py]: why do we need an all day field? they can just specify start of day to end of day in the datetime

Comment (NeilAllavarpu) [reply account/models.py]: We need to display an all day OOO separately from other OOOs - should we have a boolean all_day in the backend, or should we detect on the front end if the start and end date have the same time of day (hours/minutes)?

Comment (dhuangg) [reply account/models.py]: when you're posting the OOO, if it's all day, just set it to start/end at 12am

Comment (NeilAllavarpu) [reply account/models.py]: @jerrylinew removed all_day field

**Review by jerrylinew — APPROVED** [review]

**Pull request merged**